hack/.import-aliases

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think the `.import-aliases` should be composed of 3 parts:
```
{
# directly lifted from kubernetes/kubernetes codebase(v1.21.3)
# for reference: https://github.com/kubernetes/kubernetes/blob/ca643a4d1f7bfe34773c74f79527be4afd95bf39/hack/.import-aliases#L2-L54
...

# aliases of third-party packages
"sigs.k8s.io/mcs-api/pkg/apis/v1alpha1": "mcsv1alpha1",

# aliases of karmada packages
"github.com/karmada-io/karmada/pkg/apis/work/v1alpha1": "workv1alpha1",
"github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1": "policyv1alpha1",
"github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1": "clusterv1alpha1",
}
```
And all items in each part should be sort alphabetically.

Reply:
OK.